lib/exqlite/connection.ex

```diff
@@ -155,6 +159,7 @@ defmodule Exqlite.Connection do
             "./priv/sqlite/\#{arch_dir}/vss0"
           ]
       ```
+    * `:before_disconnect` - A function to run before disconnect, either a 2-arity fun or `{module, function, args}` with the close reason and `t:Exqlite.Connection.t/0` prepended to `args` or `nil` (default: `nil`)
```

Comment:
Need to wrap the docs.

```suggestion
    * `:before_disconnect` - A function to run before disconnect, either a 
      2-arity fun or `{module, function, args}` with the close reason and 
      `t:Exqlite.Connection.t/0` prepended to `args` or `nil` (default: `nil`)
```